packages/core/src/controllers/pairing.ts

```diff
@@ -240,10 +240,11 @@ export class Pairing implements IPairing {
     this.core.relayer.on(RELAYER_EVENTS.message, async (event: RelayerTypes.MessageEvent) => {
       const { topic, message } = event;
 
+      // Do not handle if the topic is not related to known pairing topics.
+      if (!this.pairings.keys.includes(topic)) return;
```

Comment:
What about operations that don't generate pairings ? Or under the hood everything generates a pairing? Thinking push_message and chat_invite etc